### PR TITLE
Stop requesting obsolete raid levels in pokebattler import and ignore future bosses

### LINKED ISSUE #189: Pokebattler import includes old and future bosses
- Stop reading tiers that don't exist anymore, since they haven't cleaned up tier 2
- For any pokemon at any tier verify that the id is not null, since they will include future bosses but with a null id

### PR DESCRIPTION
**Pull request by Artanicus**
This fixes #189

### CODE REVIEW

**Comment by Artanicus** [comment]
@Artanicus @Chefkeks This ought to be a fairly quick review. Fixes pokebattler import to work post-Mega

File: mods/pokebattler.php [hunk 1]
```diff
@@ -108,18 +102,17 @@
     // Raid tier array
     debug_log('Processing the following raid levels:');
     $raidlevels = array();
-    for($i = $start; $i <= $end; $i++) {
-        $rl = $i + 1;
-        if($rl == 6) $rl = 'MEGA';
-        $raidlevels[] = 'RAID_LEVEL_' . $rl;
+    foreach($get_levels as $level) {
+        if($level == 6) $level = 'MEGA';
```
Comment (Chefkeks) [inline mods/pokebattler.php:106]: if($level == 6) {
    $level = 'MEGA';
}

Comment (Chefkeks) [reply mods/pokebattler.php:106]: If you’re asking WHY?, let me just say „for a good reason...“

File: mods/pokebattler.php [hunk 2]
```diff
@@ -9,6 +9,9 @@
 // Check access.
 bot_access_check($update, 'pokedex');
 
+// Levels available for import at PokeBattler
+$levels = array('6', '5', '3', '1');
```
Comment (Chefkeks) [inline mods/pokebattler.php:13]: Maybe we should add a POKEBATTLER_RAID_LEVELS or POKEBATTLER_IMPORT_LEVELS or something like that to the default-config and get rid of this here?

**Review by Chefkeks — COMMENTED** [review]
Great improvements, just had small remarks - just see my comments

File: mods/pokebattler.php (hunk 2 above)
Comment (Artanicus) [reply mods/pokebattler.php:13]: That would mean removing the whole feature of being able to choose on the fly which ones to import? I'm pretty sure someone would miss it.

File: mods/pokebattler.php (hunk 1 above)
Comment (Artanicus) [reply mods/pokebattler.php]: Yeah, I just shuffled the lines around, happy to fix it to be more readable. :-)

File: mods/pokebattler.php (hunk 2 above)
Comment (Artanicus) [reply mods/pokebattler.php:13]: Let's revisit this one separately so we can all get a working import now. :-)

**Pull request merged**